### PR TITLE
Use https for all links

### PR DESCRIPTION
**Pull request by dns2utf8**
Hi all

This is a followup to https://github.com/gtk-rs/gtk/pull/1054

Cheers,
Stefan

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 [package]
-documentation = "http://gtk-rs.org/docs/gtk4/"
+documentation = "https://gtk-rs.org/docs/gtk4/"
 categories = ["api-bindings", "gui"]
 license = "MIT"
 description = "Rust bindings for the GTK 4 library"
 name = "gtk4"
-homepage = "http://gtk-rs.org/"
+homepage = "https://gtk-rs.org/"
```
Comment (GuillaumeGomez) [inline Cargo.toml:7]: Invalid link.

Comment (dns2utf8) [reply Cargo.toml:7]: I can't find the correct website for gtk4. Is it public already?

Comment (GuillaumeGomez) [reply Cargo.toml:7]: No idea... Ping @bilelmoussaoui

Comment (sdroege) [reply Cargo.toml:7]: gtk-rs.org is ok for now, and it was not published yet :)

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**